models/endoscopy_inbody_classification/docs/readme.md

```diff
@@ -0,0 +1,129 @@
+# Description
+A pre-trained model for the endoscopy inbody classification task.
+
+# Model Overview
+This model is trained using the SEResNet50 structure, whose details can be found in [1]. All datasets are from private samples of [Activ Surgical](https://www.activsurgical.com/). Samples in training and validation dataset are from the same 4 videos, while test samples are from different two videos.
+The [pytorch model](https://drive.google.com/file/d/14CS-s1uv2q6WedYQGeFbZeEWIkoyNa-x/view?usp=sharing) and [torchscript model](https://drive.google.com/file/d/1fOoJ4n5DWKHrt9QXTZ2sXwr9C-YvVGCM/view?usp=sharing) are shared in google drive.
```

Comment:
Suggest to add note:

Modify the path specified in configs/train.json and configs/inference.json: "bundle_root" to reflect where the models are downloaded. Under the path specified by "bundle_root" the expected directory structure is models/model.pt.

Reply:
Hi @jin-nvidia :
    Thanks for your advice! I have added a note of how to specify the model path in config files.
Thanks
BIn